tests/system/feature_store/test_feature_store.py

```diff
@@ -4860,12 +4865,18 @@ def test_parquet_filters(self, engine):
         vec = fstore.FeatureVector(
             name="test-fs-vec", features=["parquet-filters-fs.*"]
         )
+        vec.save()
+        get_offline_target = ParquetTarget(
```

Comment:
Same comment as before:
```suggestion
        target = ParquetTarget(
```